### ISSUE TITLE
NodeJS 11 support.

### ISSUE THREAD

**Issue by frederikhors** [sass/node-sass#2521]
NodeJS 11 support?

**Comment by Nothing-Works** [comment]
@frederikhors I think I got the same here, 404 not found right?

**Comment by saper** [comment]
Does it work if the binding is compiled from source?

**Comment by xzyfer** [comment]
Tracking in #2522

Seeing a compilation issue, that may just be related to our gyp config

```
 g++-4.7 '-DNODE_GYP_MODULE_NAME=libsass' '-DUSING_UV_SHARED=1' '-DUSING_V8_SHARED=1' '-DV8_DEPRECATION_WARNINGS=1' '-D_LARGEFILE_SOURCE' '-D_FILE_OFFSET_BITS=64' '-DLIBSASS_VERSION="3.5.4"' -I/home/travis/.node-gyp/11.0.0/include/node -I/home/travis/.node-gyp/11.0.0/src -I/home/travis/.node-gyp/11.0.0/deps/openssl/config -I/home/travis/.node-gyp/11.0.0/deps/openssl/openssl/include -I/home/travis/.node-gyp/11.0.0/deps/uv/include -I/home/travis/.node-gyp/11.0.0/deps/zlib -I/home/travis/.node-gyp/11.0.0/deps/v8/include -I../src/libsass/include  -fPIC -pthread -Wall -Wextra -Wno-unused-parameter -m64 -O3 -fno-omit-frame-pointer -std=gnu++1y -std=c++0x -fexceptions -frtti -MMD -MF ./Release/.deps/Release/obj.target/libsass/src/libsass/src/ast.o.d.raw   -c -o Release/obj.target/libsass/src/libsass/src/ast.o ../src/libsass/src/ast.cpp
g++-4.7: error: unrecognized command line option ‘-std=gnu++1y’
```

**Comment by rileyjsumner** [comment]
New issue popped up with node-11 install, `Failed at the node-sass@4.9.4 postinstall script`

**Comment by Artistan** [comment]
@frederikhors  -  manual compile just worked for me

**Comment by assoft** [comment]
update or reinstall node-sass and sass-loader

**Comment by Toso82** [comment]
@frederikhors  - Manual compile How thanks

**Comment by nschonni** [comment]
Node 11 support will be added with #2522 and be published as a to NPM once the CI blockers are resolved. You can subscribe to this to find out when that is done.

**Comment by xzyfer** [comment]
Released in 4.10.0
